tox.ini

```diff
@@ -2,6 +2,7 @@
 envlist = {windows}-{tests,unit,dev},tests,unit,dev,julia
 
 [testenv]
+pip_pre = true
```

Comment:
More specifically, before merging this PR we would need to remove this, and this can be removed once casadi publishes a "non-pre" release.